.github/workflows/e2e_tests.yml

```
@@ -4,139 +4,28 @@ on:
-                    "title": "Job URL",
-                    "value": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-                  }
-                ]
-              }
-            ]
+  liam-app-e2e_tests:
+    if: github.event.deployment_status.state == 'success' &&
+      contains(github.event.deployment.environment, 'Production') &&
+      contains(github.event.deployment.environment, 'liam-app') &&
+      contains(github.event.deployment_status.target_url, 'liam-app-git-main')
+    uses: ./.github/workflows/reusable_e2e.yml
```

actionlint (1.7.7)

error while parsing reusable workflow "./.github/workflows/reusable_e2e.yml": yaml: line 103: mapping values are not allowed in this context

(workflow-call)